### PR TITLE
168297257 administration card dashboard

### PR DESCRIPTION
**Pull request by d-al-ibm**
added translations
added structure and components in dashboard
added api to get organizations
fixed css for alert
added OrganizationCard component

### CODE REVIEW

File: src/components/Registration/RegistrationContainer.tsx [hunk 1]
```diff
@@ -346,13 +346,13 @@ export const RegistrationContainer = withRouter(
               <p className="pr-3 mt-5">{t("signUp.rightCol.title")}</p>
               <p className="small pr-3">{t("signUp.rightCol.text")}</p>
               <p className="small pr-2">
-                {t("signUp.rightCol.additionalInfo")}&nbsp;
+                {t("common.links.pagopa.text")}&nbsp;
```
Comment (gunzip) [inline src/components/Registration/RegistrationContainer.tsx]: pagopa -> ipa

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Please include a Pivotal story at the beginning of the PR title (see below).</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>This PR changes a total of 609 LOCs, that is more than a reasonable size of 250. Consider splitting the pull request into smaller ones.</td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 51258ec77d2c624ba48a5d70dc99bf1d9eb515dc
</p>

File: src/components/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -1,25 +1,163 @@
...
+        <Col className="info-column">
+          <p className="small px-2">
+            {t("dashboard.infoColumn.organizationRow.text")}
+          </p>
+          <p className="small px-2">
+            {t("common.links.ipa.text")}&nbsp;
+            <a
+              target="_blank"
+              rel="noopener noreferrer"
+              href="http://www.indicepa.gov.it"
```
Comment (gunzip) [inline src/components/Dashboard/Dashboard.tsx]:
```suggestion
              href="https://www.indicepa.gov.it"
```

File: src/components/Registration/RegistrationContainer.tsx [hunk 3]
```diff
@@ -346,13 +346,13 @@ export const RegistrationContainer = withRouter(
               <p className="pr-3 mt-5">{t("signUp.rightCol.title")}</p>
               <p className="small pr-3">{t("signUp.rightCol.text")}</p>
               <p className="small pr-2">
-                {t("signUp.rightCol.additionalInfo")}&nbsp;
+                {t("common.links.ipa.text")}&nbsp;
                 <a
                   target="_blank"
                   rel="noopener noreferrer"
                   href="http://www.indicepa.gov.it"
```
Comment (gunzip) [inline src/components/Registration/RegistrationContainer.tsx]:
```suggestion
                  href="https://www.indicepa.gov.it"
```

File: src/components/Dashboard/OrganizationCard.tsx [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+        <CardBody>
+          <Row>
+            {/*if user has an organization not in pre-draft status, show image section on the left*/}
+            {organizationStatus !==
+            OrganizationRegistrationStatusEnum.PRE_DRAFT ? (
+              <Col sm={2}>
+                <OrganizationCardImageSection />
+              </Col>
+            ) : null}
+            {/*if user has an organization not in pre-draft status, show organization info*/}
```
Comment (gunzip) [inline src/components/Dashboard/OrganizationCard.tsx]:
```suggestion
            {/* if the user has an organization in pre-draft status, show organization info */}
```

**Pull request merged**